src/main/java/com/suse/saltstack/netapi/calls/modules/Match.java

```diff
@@ -0,0 +1,62 @@
+package com.suse.saltstack.netapi.calls.modules;
+
+import com.google.gson.reflect.TypeToken;
+import com.suse.saltstack.netapi.calls.LocalCall;
+import com.suse.saltstack.netapi.utils.ClientUtils;
+
+import java.util.Arrays;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+/**
+ * salt.modules.match
+ */
+public class Match {
```

Comment:
BTW, @renner suggests to have at least have a documentation for the whole class with the URL link to it. Maybe Johannes will write more here.